*** Reading local file: /home/artyom/airflow//logs/example_bash_operator/also_run_this/2021-10-24T00:00:00+00:00/1.log
[2021-10-25, 23:24:06 UTC] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2021-10-24T00:00:00+00:00 [queued]>
[2021-10-25, 23:24:06 UTC] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2021-10-24T00:00:00+00:00 [queued]>
[2021-10-25, 23:24:06 UTC] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-25, 23:24:06 UTC] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-25, 23:24:06 UTC] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-25, 23:24:06 UTC] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): also_run_this> on 2021-10-24 00:00:00+00:00
[2021-10-25, 23:24:06 UTC] {standard_task_runner.py:52} INFO - Started process 2780 to run task
[2021-10-25, 23:24:06 UTC] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2021-10-24T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', '/home/artyom/.venv/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp0p6c1gut', '--error-file', '/tmp/tmpgn569cak']
[2021-10-25, 23:24:06 UTC] {standard_task_runner.py:77} INFO - Job 5: Subtask also_run_this
[2021-10-25, 23:24:06 UTC] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.also_run_this scheduled__2021-10-24T00:00:00+00:00 [running]> on host artyom-VirtualBox
[2021-10-25, 23:24:06 UTC] {logging_mixin.py:109} WARNING - /home/artyom/.venv/lib/python3.8/site-packages/airflow/models/xcom.py:322 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` flag is not set to True.  Set this flag to True if this type object's state is safe to use in a cache key, or False to disable this warning.
[2021-10-25, 23:24:07 UTC] {logging_mixin.py:109} WARNING - /home/artyom/.venv/lib/python3.8/site-packages/airflow/models/renderedtifields.py:126 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` flag is not set to True.  Set this flag to True if this type object's state is safe to use in a cache key, or False to disable this warning.
[2021-10-25, 23:24:07 UTC] {logging_mixin.py:109} WARNING - /home/artyom/.venv/lib/python3.8/site-packages/airflow/models/renderedtifields.py:162 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2021-10-25, 23:24:07 UTC] {logging_mixin.py:109} WARNING - /home/artyom/.venv/lib/python3.8/site-packages/airflow/models/renderedtifields.py:159 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` flag is not set to True.  Set this flag to True if this type object's state is safe to use in a cache key, or False to disable this warning.
[2021-10-25, 23:24:07 UTC] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-10-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-24T00:00:00+00:00
[2021-10-25, 23:24:07 UTC] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-10-25, 23:24:07 UTC] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2021-10-24T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2021-10-24 00:00:00+00:00: scheduled__2021-10-24T00:00:00+00:00, externally triggered: False>"']
[2021-10-25, 23:24:07 UTC] {subprocess.py:85} INFO - Output:
[2021-10-25, 23:24:07 UTC] {subprocess.py:89} INFO - run_id=scheduled__2021-10-24T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2021-10-24 00:00:00+00:00: scheduled__2021-10-24T00:00:00+00:00, externally triggered: False>
[2021-10-25, 23:24:07 UTC] {subprocess.py:93} INFO - Command exited with return code 0
[2021-10-25, 23:24:07 UTC] {logging_mixin.py:109} WARNING - /home/artyom/.venv/lib/python3.8/site-packages/airflow/models/xcom.py:102 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` flag is not set to True.  Set this flag to True if this type object's state is safe to use in a cache key, or False to disable this warning.
[2021-10-25, 23:24:07 UTC] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20211024T000000, start_date=20211025T162406, end_date=20211025T162407
[2021-10-25, 23:24:07 UTC] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-25, 23:24:07 UTC] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-10-25, 23:24:07 UTC] {logging_mixin.py:109} WARNING - /home/artyom/.venv/lib/python3.8/site-packages/airflow/models/dag.py:1069 SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
[2021-10-25, 23:24:07 UTC] {logging_mixin.py:109} WARNING - /home/artyom/.venv/lib/python3.8/site-packages/airflow/models/dag.py:1069 SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)


